### ISSUE TITLE
R25.03 Technical Maintenance

### ISSUE THREAD

**Issue by MaximilianHauer** [eclipse-tractusx/sig-release#855]
## Description

As part of the normal product maintenance, a couple of improvements/technical support items/needed upgrades are identified which are planned to take place.

Besides the linked issues, there are also continuous efforts ongoing to keep the products compliant with Eclipse Foundation Development Process as well as the TX Release Guidelines (e.g. security, IP checks, etc).

### Overall (Portal, IAM, Policy-Hub, SSI)

- [x] https://github.com/eclipse-tractusx/portal/issues/240
- [x] https://github.com/eclipse-tractusx/ssi-credential-issuer/issues/60
- [x] https://github.com/SAP/ssi-dim-middle-layer/issues/3
- [x] https://github.com/eclipse-tractusx/portal-backend/issues/1148
- [x] https://github.com/eclipse-tractusx/portal-shared-components/issues/399
- [x] https://github.com/eclipse-tractusx/portal-frontend/issues/1400
- [x] https://github.com/eclipse-tractusx/portal-frontend-registration/issues/326
- [x] https://github.com/eclipse-tractusx/portal-assets/issues/447
- [x] https://github.com/eclipse-tractusx/ssi-authority-schema-registry/issues/108
- [x] https://github.com/eclipse-tractusx/ssi-credential-issuer/issues/312
- [x] https://github.com/eclipse-tractusx/policy-hub/issues/241
- [x] https://github.com/eclipse-tractusx/portal-backend/issues/1155
- [x] https://github.com/eclipse-tractusx/portal-shared-components/issues/425

### Portal Frontend

- [x] https://github.com/eclipse-tractusx/portal-frontend/issues/700

## Additional information 

- [x] I'm willing to contribute to this feature

**Comment by stephanbcbauer** [comment]
@MaximilianHauer not sure if the new feature template would be helpful for this kind of tasks, but at least every feature would be described in the same manner?

**Comment by MaximilianHauer** [comment]
yes :) every issue should have a description / acceptance criteria / Test cases.

ill adjust this here tomorrow to be a bit more fitting

**Comment by stephanbcbauer** [comment]
Some hints from Release Management (@ther3sa) and Tractus-X Project Lead (@stephanbcbauer)
- Status currently in Inbox. ⇾ Only features with status backlog are considered in open planning
- Please add missing sections from the feature template, or fill them out
- Please add contributor and committer
- Please add assignee (overall responsible person who drives the feature)

**Comment by evegufy** [comment]
I move https://github.com/eclipse-tractusx/portal-backend/issues/1052 into https://github.com/eclipse-tractusx/sig-release/issues/779, as it's already done

**Comment by MaximilianHauer** [comment]
updating the release planning status to be done. open issues will be moved to the R25.06 maintenance issue
